open-source-policy/policy.md

```diff
@@ -0,0 +1,168 @@
+specify how the code may be used, modified, or shared. When users modify CISA
+code, they should review and understand the terms of the open source license in
+question.
+
+Each project may need to modify or extend the above LICENSE and CONTRIBUTING
+files as needed for its own circumstances.
+
+## Distribution of Code ##
+
+There is a misconception that FOSS that is distributed to the public should not
+be integrated or modified for use in sensitive systems. On the contrary, FOSS is
+often preferred for use in sensitive systems, due in part to its increased
+auditability. In other words, security in FOSS must be designed never to rely on
+obscurity in how the code works.
+
+In addition, while open source licenses permit the user to modify FOSS for
+internal use without obligating them to distribute source code to the public,
+when the user chooses to distribute the modified FOSS outside the user's
+organization, then the code is subject to whatever license it carries.
+
+## Exceptions ##
+
+The only conditions where code shall not be developed and released in the open
+are:
+
+* The U.S. Government does not have the rights to reproduce and release the
+  item.
+
+* The public release of the item is restricted by other law or regulation, such
+  as the Export Administration Regulations or the International Traffic in Arms
+  Regulation.
+
+These decisions will be made as needed by the CISA Infrastructure team, which
+will lead an interdisciplinary team to review the conditions under which code
+will not be made available publicly. Any further exemptions will be rare,
+documented publicly, and the result of compelling interest.
+
+If an existing solution cannot be found in the open source community, CISA may
+consider other options, including creating an open source solution itself.
+Ultimately, the software that best meets the needs and mission of CISA should be
+used.
+
+## Further Reading ##
+
+[OMB M-16-21](https://sourcecode.cio.gov/) - A White House policy pertaining to
+federal source code, published by the White House Office of Management and
+Budget. M-16-21 directs all agencies to publish some custom developed code as
+open source code, to update acquisition requirements to capture code developed
+by vendors for GSA, and to inventory [all open- and closed-source
+code](https://open.gsa.gov/code.json).
+
+[GSA Open Source Policy](https://open.gsa.gov/oss-policy/) - GSA's agency-wide
+open source policy, created based on OMB's M-16-21 policy.
+
+## Thanks ##
+
+CISA would like to thank 18-F, Consumer Financial Protection Bureau, Department
+of Defense, and Office of Management and Budget for their work in blazing the
+path for the use of FOSS in the Federal Government.
+
+## Future Changes ##
+
+This policy is a living document. CISA expects to make changes to this policy in
+the future, and we welcome
+[issues](https://github.com/cisagov/development-guide/issues) and pull requests.
```

Comment:
Do you want "pull requests" to be a link as well?

Reply:
Good idea!